core/src/block.rs

```diff
@@ -133,7 +133,7 @@ pub struct PendingBlock {
 }
 
 /// Builder for `PendingBlock`
-pub struct BlockBuilder {
+pub struct BlockBuilder<'a> {
```

Comment:
Doesn't clippy complain about one letter lifetime name? I'd better name it `world` as it's done in `wasm` module